src/foam/classloader/NodeModelFileDAO.js

```diff
@@ -23,20 +23,40 @@ foam.CLASS({
   properties: [
     {
       name: 'classpath'
+    },
+    {
+      name: 'loadedModels',
```

Comment:
Context.arequire() does the check.

https://github.com/foam-framework/foam2/blob/master/src/foam/classloader/ClassLoader.js#L78

It's actually more complicated than just caching the loaded models.  Cause we also do special bookkeeping to resolve circular dependencies.